### PR TITLE
Added tests for ResNet-50 and SqueezeNet v1.1

### PR DESCRIPTION
**Pull request by arrybn**
**Merge with extra**: https://github.com/opencv/opencv_extra/pull/350

### This pullrequest changes
Added tests for ResNet-50 and SqueezeNet v1.1

### CODE REVIEW

File: modules/dnn/test/test_caffe_importer.cpp [hunk 1]
```diff
@@ -157,4 +157,35 @@ TEST(Reproducibility_SSD, Accuracy)
...
+    Mat out = net.forward();
+
+    Mat ref = blobFromNPY(_tf("resnet50_prob.npy"));
+    normAssert(ref, out);
+}
+
+TEST(Reproducibility_SqueezeNet_v1_1, Accuracy)
+{
+    Net net = readNetFromCaffe(findDataFile("squeezenet_v1.1.prototxt", false),
+                               findDataFile("squeezenet_v1.1.caffemodel", false));
```
Comment (alalek) [inline modules/dnn/test/test_caffe_importer.cpp]: `dnn/` prefix is required.

**Comment by alalek** [comment]
Please update name in `test_halide_nets.cpp` too:
```.diff
-squeezenet_v1_1.caffemodel
+squeezenet_v1.1.caffemodel
```

File: modules/dnn/test/test_caffe_importer.cpp (hunk 1 above)
Comment (arrybn) [inline modules/dnn/test/test_caffe_importer.cpp]: fixed, thanks

**Comment by arrybn** [comment]
@alalek, I updated names for model's files in test_halide_nets.cpp

**Comment by alalek** [comment]
:+1:

**Pull request merged**